scripts/run_mypy_checks.py

```diff
@@ -66,12 +66,15 @@ def get_mypy_cmd(files: Optional[List[str]]) -> List[str]:
     mypy_cmd = 'mypy'
 
     if files:
-        cmd = [mypy_cmd, '--config-file', CONFIG_FILE_PATH] + files
+        cmd = [
+            mypy_cmd, '--install-types', '--non-interactive', '--config-file',
```

Comment:
I wonder if --install-types is deterministic. Would it be better to add the [types-* packages](https://mypy-lang.blogspot.com/2021/06/mypy-0900-released.html) manually to requirements_dev.in and keep the command as it was?

Otherwise there's a risk that some type lib breaks things in the future since we aren't pinning the versions of the stubs. It also can result in different devs having different envs which makes problems harder to debug.

This is all assuming that mypy raises an error if a needed stub is not present -- is that the case currently?